.flowconfig

```diff
@@ -31,3 +31,5 @@ module.system.node.resolve_dirname=static/src/javascripts
 module.system.node.resolve_dirname=static/src/javascripts-legacy
 module.system.node.resolve_dirname=static/vendor/javascripts
 module.system.node.resolve_dirname=node_modules
+
+unsafe.enable_getters_and_setters=true
```

Comment:
Interesting to note that Flow [always considers getters and setters to be unsafe](https://github.com/facebook/flow/commit/6dea70976b4519a2e0fcc69977eed8699ea66243) and side-effecty.